game-service/src/main/java/org/libertybikes/game/core/GameMap.java

```diff
@@ -0,0 +1,122 @@
+package org.libertybikes.game.core;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class GameMap {
+
+    int[][] playerPositions = null;
+    ArrayList<Obstacle> obstacles = new ArrayList<Obstacle>();
+    ArrayList<MovingObstacle> movingObstacles = new ArrayList<MovingObstacle>();
+
+    public GameMap() {
+        int mapNum = 4;
+        int randomNum = ThreadLocalRandom.current().nextInt(1, mapNum + 1);
+        String methodName = "map" + randomNum;
+        try {
+            Class<?> c = Class.forName("org.libertybikes.game.core.GameMap");
+            Method method = c.getDeclaredMethod(methodName);
```

Comment:
I think a normal switch/case statement would be sufficient here instead of reflection